static/js/runtime-config.js

```diff
@@ -4,7 +4,18 @@
 export default class RuntimeConfig {
   constructor (initialConfig = {}) {
     this._data = { ...initialConfig };
-    this._onUpdateListener = () => {};
+    
+    /**
+     * Hold listeners that applies to all keys
+     * @type {Object[]}
+     */
+    this._generalListeners = [];
+    
+    /**
+     * Hold listeners that applies to specific keys
```

Comment:
super nit but maybe we remove the word Hold? (unrelated but I think it should just be apply not applies)
so maybe `Listeners that apply to specific keys`, and same with generalListeners?